### PR TITLE
Api server 2

### PR DESCRIPTION
**Pull request by pavel-kokolemin**

### CODE REVIEW

**Review by Ation — APPROVED** [review]

File: BlocksettleNetworkingLib/SslServerConnection.cpp [hunk 1]
```diff
@@ -188,10 +191,8 @@ int SslServerConnection::callback(lws *wsi, int reason, void *user, void *in, si
             if (clientIt != clients_.end()) {
                SPDLOG_LOGGER_DEBUG(logger_, "force close client {}", bs::toHex(clientId));
                auto clientWsi = clientIt->second.wsi;
-               if (clientWsi) {
-                  lws_close_reason(clientWsi, LWS_CLOSE_STATUS_PROTOCOL_ERR, nullptr, 0);
-                  lws_set_timeout(clientWsi, PENDING_TIMEOUT_USER_OK, LWS_TO_KILL_SYNC);
-               }
+               lws_close_reason(clientWsi, LWS_CLOSE_STATUS_PROTOCOL_ERR, nullptr, 0);
+               lws_set_timeout(clientWsi, PENDING_TIMEOUT_USER_OK, kForcedDisconnectTimeSeconds);
```
Comment (Ation) [inline BlocksettleNetworkingLib/SslServerConnection.cpp:195]: https://github.com/warmcat/libwebsockets/issues/1463

**Review by Ation — CHANGES_REQUESTED** [review]
Previous implementation looks correct. Actually it is how it was done in the first place in WsServerConnection

File: BlocksettleNetworkingLib/SslServerConnection.cpp (hunk 1 above)
Comment (pavel-kokolemin) [reply BlocksettleNetworkingLib/SslServerConnection.cpp:195]: if we use `LWS_TO_KILL_SYNC` here, server will not send pending packets, for example with failed login error message

**Comment by pavel-kokolemin** [comment]
> Previous implementation looks correct. Actually it is how it was done in the first place in WsServerConnection

WsServerConnection is broken too, it won't send pending packets.

**Pull request merged**